src/app/search/loadouts/search-filters/freeform.ts

```diff
@@ -88,6 +92,67 @@ const freeformFilters: FilterDefinition<
       };
     },
   },
+  {
+    keywords: 'contains',
+    description: tl('LoadoutFilter.Contains'),
+    format: 'freeform',
+    suggestionsGenerator: ({ d2Definitions, loadouts, selectedLoadoutsStore }) => {
+      if (!d2Definitions || !loadouts) {
+        return [];
+      }
+
+      return loadouts.flatMap((loadout) => {
+        if (loadout.classType !== selectedLoadoutsStore?.classType) {
+          return [];
+        }
+
+        const itemSuggestions = loadout.items.map((item) => {
+          const resolutionInfo = getResolutionInfo(d2Definitions, item.hash);
+          const definition =
+            resolutionInfo && d2Definitions.InventoryItem.getOptional(resolutionInfo.hash);
+          return (
+            definition &&
+            `contains:${quoteFilterString(definition.displayProperties.name.toLowerCase())}`
```

Comment:
Interestingly this kind of immediately obsoletes the `subclass:voidwalker` search, since `contains:voidwalker` also works.

Reply:
Yeah, the main reason for `subclass:` is probably for the "community name" type of suggestions. The element name is one example, but if we ever do things like "well" or "tether" then it would have more value. Not sure if it matters personally whether we have the subclass names in both or not. Thoughts?

Reply:
Yeah just a thought. I think the other aliases (element, etc) plus community names are really useful.